### ISSUE TITLE
Admin : Develop the profile page of an administrator account

### ISSUE THREAD

**Issue by t-bello7** [Ingressive-for-Good/I4G-OPENSOURCE-FRONTEND-PROJECT-2022#169]
## Title 
Develop the profile page of an administrator account

## Summary 
- Create the profile page based on Figma Design.
- Add routes for the profile page

## Additional Information 
- Kindly follow the file structure

**Comment by ScottDeev** [comment]
Trying to find the design on figma but I'm lost. Can a screenshot of the page be dropped please?

**Comment by Osilaja78** [comment]
@ScottDeev @t-bello7 i think this is the page. If it is i would love to work on it, since @ScottDeev already has another issue he is working on. Need to know before i assign myself.

![Screenshot 2022-10-21 162116](https://user-images.githubusercontent.com/105312707/197232525-9faf5c18-3385-4e09-b29b-08e9d9396f56.png)

**Comment by t-bello7** [comment]
Sure you can. Scott hasn't been assigned to it.

**Comment by Osilaja78** [comment]
Alright

**Comment by Osilaja78** [comment]
i just discovered that the page has already been done by someone else : (

![Capture](https://user-images.githubusercontent.com/105312707/197371250-eefc0d1a-8a75-461f-9616-659d41779c15.PNG)

**Comment by dprincecoder** [comment]
@t-bello7 so this issue is redundant?

**Comment by Osilaja78** [comment]
Yes, I think it's better you close it so it won't get another person working on something that has already been done.
